### DISCUSSION TITLE
Trying to add a page but getting a page not found error?

### DISCUSSION THREAD

**Discussion by 9AMTech** [gatsbyjs/gatsby#38105] [Help]
![image](https://github.com/gatsbyjs/gatsby/assets/48345582/95ff0e9a-826e-422f-b6f0-b72f4cefed31)
![image](https://github.com/gatsbyjs/gatsby/assets/48345582/0b092a68-974f-41f7-9190-dcded54e00d8)

I'm new to Gatsby and I've been following the docs and various guides pretty closely to try and build out my first Gatsby app, but it seems like this error is a bug. I have made other pages as seen in the Nav component and they all work as expected, for some reason my About page doesn't want to work despite having nearly the same code as another page. 

**Nav.tsx**
````
import React from "react";
import { Link } from "gatsby";
import Button from "../Button";
import CartPNG from "../../images/cart.png";

const Nav = () => {
  return (
    <nav>
      <Link to="/"> Home </Link>
      <Link to="/products">Products</Link>
      <Link to="/about">About</Link>
			
      <Button content={<img src={CartPNG} alt="Cart" />} />
    </nav>
  );
};

export default Nav;
````
**About.tsx**

```
import React from "react";
import { HeadFC } from "gatsby";
import Layout from "../components/Layout/Layout";
import Paragraph from "../components/Paragraph";

const AboutPage = () => {
  return (
    <Layout>
      <Paragraph
		containerStyle=""
		textStyle=""
		text="Hahaa!"
       />
    </Layout>
  )
};

export default AboutPage;

export const Head: HeadFC = () => <title>About</title>;
```

**Comment by 808vita** [comment]
restarting will probably fix this . do npm start again after existing the current .
